test/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { Connection } from '../src/db/Connection';
+
+const dbUrl = process.env.DATABASE_URL || 'mongodb://localhost:27017/measure-repository';
```

Comment:
Should this default to the same one in `.env.test`? We don't want to accidentally use the real database during tests

```suggestion
const dbUrl = process.env.DATABASE_URL || 'mongodb://localhost:27017/measure-repository-test';
```